### DISCUSSION TITLE
Worth mentioning : CSS Isolation support is mandatory in project as of version 7.0

### DISCUSSION THREAD

**Discussion by NonoSparc** [Blazored/Modal#440] [General]
Hi. I just upgraded an old project from 6.01 to 7.0 and the result was ugly. No styles were applied to dialog shown.

It happens that my project was not making use of CSS isolation at all so this line was missing in index.html:

```
<head>
...
    <link href="MyBlazorApp.Client.styles.css" rel="stylesheet" />
...
</head>
```

Adding it solved my issue, but I think it would be worth mentioning it in the Getting Started page of the documentation.

**Comment by chrissainty** [comment]
This is a great point, thank you for raising it @NonoSparc. I've created an issue for this.
